sweep/interface.go

```diff
@@ -0,0 +1,29 @@
+package sweep
+
+import (
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/lnwallet"
+)
+
+// Wallet contains all wallet related functionality required by sweeper.
+type Wallet interface {
+	// PublishTransaction performs cursory validation (dust checks, etc) and
+	// broadcasts the passed transaction to the Bitcoin network.
+	PublishTransaction(tx *wire.MsgTx) error
+
+	// ListUnspentWitness returns all unspent outputs which are version 0
+	// witness programs. The 'minconfirms' and 'maxconfirms' parameters
+	// indicate the minimum and maximum number of confirmations an output
+	// needs in order to be returned by this method. Passing -1 as
```

Comment:
Here is the confusing minConf comment. Does passing 0 mean unconfirmed or confirmed outputs? 😛 

Probably best to pass 1, and just test that actually works.

Reply:
Ah ok. That was copy pasted. I removed the story about `-1`. It seems to be a left over from old code.